applications/app/controllers/MediaController.scala

```diff
@@ -37,6 +37,7 @@ class MediaController(contentApiClient: ContentApiClient) extends Controller wit
     val response: Future[ItemResponse] = contentApiClient.getResponse(
       contentApiClient.item(path, edition)
         .showFields("all")
+        .showBlocks("main")
```

Comment:
what are main blocks and how big are they?  I only dealt with body blocks before, and the problem there is we would fetch the body twice, once in the  blocks and once in the body field.  My worry here is you are adding more data to every response (for media) so we need to think about cost and whether we need to scale things up.

For live blogs (which have massive bodys) I got around that with a horrible hack that if we're really sure it is a live blog we only get the blocks, otherwise we only get the body.  This does mean some old live blogs don't really format properly as it now tries to render them as articles.